packages/angular/projects/ui-angular/src/lib/primitives-poc/password-field/password-field-directive/password-field.directive.ts

```diff
@@ -0,0 +1,56 @@
+import { Directive, HostBinding, Input } from '@angular/core';
+import { nanoid } from 'nanoid';
+
+const $Directive_Selector = '[amplify-password]';
+@Directive({
+  selector: $Directive_Selector,
+})
+export class PasswordDirective {
```

Comment:
Idea: since `PasswordDirective` is an `InputDirective` but with more password capability, what do you think about having it extend `InputDirective`?

Reply:
Done.

Reply:
I don't think this is done yet, but we can punt this when we have this merged and have primitives altogether.